spec.md

```diff
@@ -1369,6 +1376,49 @@ The CO MUST implement the specified error recovery behavior when it encounters t
 | Invalid `starting_token` | 10 ABORTED | Indicates that `starting_token` is not valid. | Caller SHOULD start the `ListSnapshots` operation again with an empty `starting_token`. |
 
 
+#### `ControllerExpandVolume`
```

Comment:
When is this safe to call by the CO? Should it be called before or after `ControllerPublishVolume`? Could you elaborate when the CO can call this RPC safely through volume lifecycle? And say if the plugin also has the `EXPAND_VOLUME` *node* capability, what's the change in volume lifecycle between `ControllerExpandVolume` and `NodeExpandVolume`?

Reply:
fixed